addons/sourcemod/translations/shavit-wr.phrases.txt

```diff
@@ -202,4 +202,15 @@
 	{
 		"en"		"Time"
 	}
+	// ---------- Messages ---------- //
+	"WRStageTimeAhead"
+	{
+		"#format"	"{1:s},{2:s},{3:d},{4:s},{5:s},{6:s},{7:s},{8:s},{9:s},{10:s}"
+		"en"		"{1}You have reached stage {2}{3}{4} with a time of {5}{6}{7} (WR {8}{9}{10})."
```

Comment:
(WR {8}-{9}{10})

Reply:
Just a cosmetic.

Reply:
should do `Stage #0 reached @ 00:00.00 (WR +00:00.00) (00 jumps & 00 strafes)` and combine translations strings WRStageTimeAhead & WRStageTimeBehind

Reply:
> should do `Stage #0 reached @ 00:00.00 (WR +00:00.00) (00 jumps & 00 strafes)` and combine translations strings WRStageTimeAhead & WRStageTimeBehind

I combined the translations into one called WRStageTime